readthedocs/organizations/models.py

```diff
@@ -93,7 +94,23 @@ def get_absolute_url(self):
 
     @property
     def users(self):
-        return (self.members.all() | self.owners.all().distinct()).distinct()
+        sso_users = User.objects.none()
+        # TODO: find another way to check if we are under corporate site
```

Comment:
Can't we just override this on the .com?

Reply:
I think we don't have a pattern to override a method from a Model.

The only way I'm thinking now (since this is a queryset method) could be move this method into the `OrganizationQuerySet` and use `SettingsOverrideObject` in that class. Makes sense?

Reply:
It will be weird, though. It will work like `Organization.objects.users(organization=slug)` but it will return `User` instances instead of `Organization` what is what I'd expect.

Reply:
What is this logic even used for? I really don't like having a `readthedocsinc` import in this codebase, it feels really janky. We should probably just change the logic where this is used to call code that isn't in this codebase.

Reply:
It's used in,

- `AdminPermissionBase.is_member` (`obj` could be an Organization)
- `CorporateAdminPermission.is_member`
- old migrations (oauth.0003_move_github)
- corporate profile details

I'm tempted to remove this `users` property together with `is_member` (we talked about the latter already)

Reply:
OK. I moved this code to `CorporateAdminPermission.is_member`, where it seems it's the correct place for this code. Then, when we need to check for membership, we should use that classmethod.

I removed these `users` and `member` properties to avoid confusions (they won't return SSO members). We could re-add them later if we need them.

Reply:
> I removed these `users` and `member` properties to avoid confusions (they won't return SSO members)

I finally moved them into the `AdminPermission` class, so we can override them from corporate. Corporate requires the `Organization.members` method.